MonoGame.Framework/Quaternion.cs

```diff
@@ -786,10 +786,10 @@ public static void Slerp(ref Quaternion quaternion1, ref Quaternion quaternion2,
 		    }
 		    else
 		    {
-		        float num5 = (float) Math.Acos((double) num4);
-		        float num6 = (float) (1.0 / Math.Sin((double) num5));
```

Comment:
Precision lost!
The division (reciprocal) was calculated with double precision.